curriculum/challenges/english/25-front-end-development/quiz-basic-html/66df3b712c41c499e9d31e5b.md

```diff
@@ -1373,3 +1373,447 @@ Which of the following elements should NOT be placed inside of the `head`?
+
+#### --answer--
+
+ID names cannot contain underscores and must be unique.
+
+### --question--
+
+#### --text--
+
+Which of the following statements about HTML heading elements is true?
+
+#### --distractors--
+
+The text size increases from `<h1>` to `<h6>`.
+
+---
+
+The text size remains the same for all heading elements.
+
+---
+
+`<h3>` is larger than `<h2>`.
+
+#### --answer--
+
+The text size decreases from `<h1>` to `<h6>`.
+
+### --question--
+
+#### --text--
+
+Which statement is incorrect about HTML elements?
+
+#### --distractors--
+
+HTML elements are the basic building blocks of a webpage.
+
+---
+
+Some codebases include a forward slash (/) inside void elements.
+
+---
+
+A boolean attribute is one that works simply by being present or absent in a tag.
+
+#### --answer--
+
+Void elements contain content and only have a start tag, like `<img>` and `<meta>`.
+
+### --question--
+
+#### --text--
+
+Which of the following statements about void elements in HTML is true?
+
+#### --distractors--
+
+A void element must always include a forward slash / before closing.
+
+---
+
+A void element should never include a forward slash /.
+
+---
+
+The forward slash / is mandatory for self-closing void elements in all versions of HTML.
+
+#### --answer--
+
+Both `<img>` and `<img/>` are acceptable in modern HTML.
+
+### --question--
+
+#### --text--
+
+What is the primary purpose of comments in programming?
+
+#### --distractors--
+
+To make the code run faster.
+
+---
+
+To change how the browser displays content.
+
+---
+
+To store temporary data within the code.
+
+#### --answer--
+
+To leave notes for yourself and other developers in the code.
+
+### --question--
+
+#### --text--
+
+Which statement about HTML heading elements is correct?
+
+#### --distractors--
+
+
+`<h6>` is the most important heading element.
+
+---
+
+All heading elements `<h1>` to `<h6>` have the same importance.
+
+---
+
+`<h3>` is more important than `<h1>`.
+
+#### --answer--
+
+`<h1>` has the highest importance, and <h6> has the least importance.
+
+### --question--
+
+#### --text--
+
+What is the purpose of the `<em>` element in HTML?
+
+#### --distractors--
+
+To make text bigger.
+
+---
+
+To change the text color.
+
+---
+
+To underline words for emphasis.
+
+#### --answer--
+
+To highlight important text by making it italic.
+
+### --question--
+
+#### --text--
+
+What does the `<strong>` element do in HTML?
+
+#### --distractors--
+
+---
+
+Increases the font size.
+
+---
+
+Changes the text color.
+
+---
+
+Underlines the text.
+
+#### --answer--
+
+Makes text bold to show importance.
+
+### --question--
+
+#### --text--
+
+What is the main difference between the `<figure>` and `<figcaption>` elements in HTML?
+
+#### --distractors--
+
+`<figure>` is used for captions, while `<figcaption>` is used for images.
+
+---
+
+`<figcaption>` is used to display images, while `<figure>` provides a caption.
+
```

Comment:
these should be changed because they basically say the same thing. we want the all options to be different from each other.